### ISSUE TITLE
Wormhole gateway chain `wormchain-1` archive node sync breaks on block `4411563`

### ISSUE THREAD

**Issue by clemensgg** [wormhole-foundation/wormhole#3943]
## Description and context
After successfully syncing over 3M blocks from the genesis block, a wormhole gateway `wormchain-1` archive node sync breaks on block `4411563` with an app-hash error using the version history from [the provided syncing guide](https://github.com/wormhole-foundation/wormhole/blob/main/wormchain/syncing.md).

## Steps to reproduce
1. Follow steps [in the wormchain mainnet syncing guide](https://github.com/wormhole-foundation/wormhole/blob/main/wormchain/syncing.md)

## Experienced behavior
A `wormchaind-v2.18.1` archive node apphashes during block-sync on block `4411563`:
```
4:47PM INF indexed block exents height=4411563 module=txindex
4:47PM ERR Error in validation err="wrong Block.Header.AppHash.  Expected E74E1453A45DC2F427A36765
9BA5B9DD56DE6CC48ABF3244E2348AF7FC5AEC06, got CC32E562E6C9C2EF4D0BB7ABCE7B6AB82C2E452E1B2475557C1237A314EEC978" module=
blockchain
```

```
name: wormchain
server_name: <appd>
version: v2.18.1
commit: 821d66eae
build_tags: ledger
go: go version go1.20.2 linux/amd64
```

## Expected behavior
`wormchaind-v2.18.1` should be able to sync past block `4411563`

**Comment by nik-suri** [comment]
I've been digging into this and I believe the syncing instructions might be out of date. When your node starts apphashing at block `4411563`, could you please stop the sync and update to `wormchaind-v2.18.1.1`?

Then, when your node stops at block `4449129`, follow the instructions as written to upgrade to `wormchaind-v2.23.0`

**Comment by clemensgg** [comment]
thank you @nik-suri, i have updated the syncing instructions to reflect the `wormchaind-v2.18.1.1` and recent `wormchaind-v2.24.2-wormchaind` upgrades. Please review and feel free to merge.

**Comment by clemensgg** [comment]
Hi @nik-suri, unfortunately starting after rollback on `v2.18.1.1` from block `4411562` results in db corruption:
```
~$ tail -f rollback.log 
Rolled back state to height 4411562 and hash C2458ACCAA16AD357AF8B50DF92D7C583C6B7AC1FF4D9C71DDDE033E892063A0^C
~$ ./wormchaind-v2.18.1.1 start --home ~/.wormchaind/ > wormchain.log 2>&1 & && tail -f wormchain.log 
10:59PM INF starting node with ABCI Tendermint in-process
10:59PM INF service start impl=multiAppConn module=proxy msg={}
10:59PM INF service start connection=query impl=localClient module=abci-client msg={}
10:59PM INF service start connection=snapshot impl=localClient module=abci-client msg={}
10:59PM INF service start connection=mempool impl=localClient module=abci-client msg={}
10:59PM INF service start connection=consensus impl=localClient module=abci-client msg={}
10:59PM INF service start impl=EventBus module=events msg={}
10:59PM INF service start impl=PubSub module=pubsub msg={}
10:59PM INF service start impl=IndexerService module=txindex msg={}
10:59PM INF ABCI Handshake App Info hash="�E�̪\x16�5z��\r�-|X<kz��M�q��\x03>� c�" height=4411562 module=consensus protocol-version=0 software-version=v2.18.1.1
10:59PM INF ABCI Replay Blocks appHeight=4411562 module=consensus stateHeight=4411562 storeHeight=4411563
10:59PM INF Replay last block using real app module=consensus
10:59PM INF executed block height=4411563 module=consensus num_invalid_txs=0 num_valid_txs=16
panic: version 4411563 was already saved to different hash 36BB9CAC01D6C17B02094730E96D0B9C128625D8F5E5E20E634E2C5A771CF488 (existing hash 14A43EB15DA277822D329E6F4115FDF9654E92F71DF10E50EC78FC5F81BC617A)
```

**Comment by clemensgg** [comment]
solved by setting `--halt-height 4411562` on `v2.18.1`
